lua/swamp/chat/sv_commands/bounties.lua

```diff
@@ -177,3 +177,42 @@ end, {
     global = false,
     throttle = false
 })
+
+--Could be it's own file, but eh
+RegisterChatCommand({'givepointsrandom', 'randomgivepoints'}, function(ply, arg)
+    local p = math.floor(tonumber(arg) or 0)
+
+    if p > 0 then
+        local ranply = {}
+
+        for k, v in ipairs(player.GetHumans()) do
+            if v:IsActive() and v ~= ply then
+                table.insert(ranply, v)
+            end
+        end
+        
+        if #ranply == 0 then return end
+        
+        local rcvr = ranply[math.random(#ranply)]
+        
+        ply:TryTakePoints(p, function()
+            if not IsValid(ply) then return end
+
+            if IsValid(rcvr) then
+                rcvr:GivePoints(p, function()
+                    ply:Notify("You gave " .. rcvr:Nick() .. " " .. p .. " points.")
+                    rcvr:Notify(ply:Nick() .. " gave you " .. p .. " of their points.")
+                end,
+                function() return end)
```

Comment:
you don't need a fail callback if its empty, this just pollutes the code. you don't need to use a callback at all here, just do GivePoints(p)